### PR TITLE
Documents supported meta fields

### PR DESCRIPTION
**Pull request by wtekeu**
https://github.com/medic/medic/issues/5768

### CODE REVIEW

File: configuration/forms.md [hunk 1]
```diff
@@ -556,3 +556,18 @@ Note that `FORM_CODE` should be replaced with the form code as defined in the JS
 
 ### Automatic changes with XLSForms
 If using a [XLSForm](http://xlsform.org/) and using [`medic-conf`](https://github.com/medic/medic-conf) to convert to XForm, the necessary fields will be automatically added to the resulting XForm. You can override the default prefix and separator by declaring `sms_keyword` and `sms_seperator` respectively in the Settings tab.
+
+## Supported XLSForm Meta Fields
+The following meta elements are supported:
+| element | type | description | value |
```
Comment (newtewt) [inline configuration/forms.md]: The table markdown isn't being applied properly when I view the file in github. Adding a newline between lines 561 and 562 fixes it. I can commit that change if you'd like.

File: configuration/forms.md [hunk 2]
```diff
@@ -556,3 +556,18 @@ Note that `FORM_CODE` should be replaced with the form code as defined in the JS
...
+|---|---|---|---|
+| `timeStart` | datetime | A timestamp of when the form entry was started. | now() |
+| `timeEnd` | datetime | A timestamp of when the form entry ended. | now() |
+
+These meta elements have corresponding `<bind>` elements with either a calculation or with `preload attributes`.
+
+Supported preload attribute combinations are:
+| jr:preloadParams | jr:preload | value |
```
Comment (newtewt) [inline configuration/forms.md]: Same issue here with the table not being applied.

File: configuration/forms.md (hunk 1 above)
Comment (wtekeu) [reply configuration/forms.md]: Please do.

File: configuration/forms.md (hunk 2 above)
Comment (wtekeu) [reply configuration/forms.md]: Again, Please do

**Pull request merged**